src/main/java/fr/xephi/authme/commands/AdminCommand.java

```diff
@@ -97,7 +103,11 @@ public boolean onCommand(CommandSender sender, Command cmnd, String label,
 
         if (args[0].equalsIgnoreCase("purge")) {
             if (args.length != 2) {
-                sender.sendMessage("Usage: /authme purge <DAYS>");
+                sender.sendMessage("Usage: /authme purge <days>");
+                return true;
+            }
+            if (Integer.parseInt(args[1]) < 30) {
```

Comment:
I disagree with that. I mean, if you need to clear database before 30 days you can just delete your database straight. Why would you purge everything that was created before 30 days?
It's kind of a safe hard limit, where you can't go under.
I still would suggest to keep 30 days for the client and 1 week-1 day for the console.